src/app/components/DataLoader/DataLoader.jsx

```diff
@@ -12,19 +12,48 @@ class DataLoader extends React.Component {
         expression: /\/research\/collections\/shared-collection-catalog\/bib\/([cp]?b\d*)/,
         pathType: 'bib',
       },
+      {
+        expression: /\/research\/collections\/shared-collection-catalog\/search\?(.*)/,
+        pathType: 'search',
+      }
     ];
     this.pathType = null;
     this.routes = {
       bib: {
-        apiRoute: matchData => `${appConfig.baseUrl}/api/bib?bibId=${matchData[1]}`,
+        apiRoute: matchData => `${props.next ? 'http://localhost:3001' : ''}${appConfig.baseUrl}/api/bib?bibId=${matchData[1]}`,
```

Comment:
Yeah, this is not something you'd eventually want to commit. I know it's a WIP but having `localhost` in the code doesn't seem like a good pattern to use.